src/main/webapp/app/overview/course-exams/course-exams.component.ts

```diff
@@ -39,6 +53,38 @@ export class CourseExamsComponent implements OnInit, OnDestroy {
             this.courseCalculationService.updateCourse(course);
             this.course = this.courseCalculationService.getCourse(this.courseId);
         });
+
+        this.studentExamTestExamUpdateSubscription = this.examParticipationService
+            .loadStudentExamsForTestExamsPerCourseAndPerUserForOverviewPage(this.courseId)
+            .subscribe((response: StudentExam[]) => {
+                this.studentExams = response!;
+            });
+
+        if (this.course!.exams) {
+            // The Map is ued to store the boolean value, if the attempt-List for one Exam has been expanded or collapsed
+            this.expandAttemptsMap = new Map(this.course!.exams!.filter((exam) => exam.testExam && this.isVisible(exam)).map((exam) => [exam.id!, false]));
+
+            // Loading the exams from the course
+            this.realExamsOfCourse = this.course!.exams!.filter((exam) => this.isVisible(exam) && !exam.testExam).sort((se1, se2) => {
+                if (se1.startDate! > se2.startDate!) {
+                    return 1;
+                }
+                if (se1.startDate! < se2.startDate!) {
+                    return -1;
+                }
+                return 0;
+            });
+
+            this.testExamsOfCourse = this.course!.exams!.filter((exam) => this.isVisible(exam) && exam.testExam).sort((se1, se2) => {
+                if (se1.startDate! > se2.startDate!) {
+                    return 1;
+                }
+                if (se1.startDate! < se2.startDate!) {
+                    return -1;
+                }
+                return 0;
+            });
```

Comment:
Seems like you're using the same sort function twice here. You could extract it into a static function to reduce duplication.
Also, what is the type of ``startDate``? Is it dayjs? If yes, I'd recommend to use the ``isAfter`` or ``isBefore`` methods